app/assets/stylesheets/preact/onboarding-modal.scss

```diff
@@ -109,11 +109,19 @@
   width: 100%;
   border-bottom: 2px solid var(--box-darker);
 
+  &.intro-slide {
```

Comment:
It's not documented yet, but consider to start using the CSS utility classes that @ludwiczakpawel has created. For reference see https://github.com/thepracticaldev/dev.to/blob/master/app/assets/stylesheets/config/_generator.scss.

Reply:
@nickytonline Hm, I see what you mean, but without documentation I am not sure how to go about using it. Do you think this could be done in a separate PR, once it has been documented? Alternatively, can you point me to a place where we actually already use it in the app so I can figure out how to apply it?

Reply:
Fair point about the documentation. Well played, well played. 😄 @ludwiczakpawel, this is probably a good task to do once we hit the next cooldown period.

@vaidehijoshi, it's still fairly new, so it's used in a few places, but only certain utility classes. I think it's fine to do this in a future PR.

Reply:
I'd love to use and apply the new utility classes; cooldown does seem like a great time for me to practice using them ! 😄